include/boost/beast/http/impl/read.ipp

```diff
@@ -112,72 +126,61 @@ operator()(
     error_code ec,
     std::size_t bytes_transferred)
 {
-    switch(state_)
+    size_t consumed = 0;
```

Comment:
You don't **have** to put this hear, you could put it in a braced scope where it is used:
```
{
    auto const consumed = p_.put(b_.data(), ec);
    total_bytes_consumed_ += consumed;
    b_.consume(consumed);
}
```

Reply:
Should be `std::size_t`